data/ca/retired/Todd-Gloria-1465de6a-ec39-4986-b3e4-e30ef7a47c20.yml

```diff
@@ -1,20 +0,0 @@
-id: ocd-person/1465de6a-ec39-4986-b3e4-e30ef7a47c20
-name: Todd Gloria
```

Comment:
I did that manually. My bad. 

Am trying to avoid going clean slate again (go back to main and run scripts from scratch), but the downside of that is that as scripts evolve, it's harder to tell which things were right or wrong.

I can fix all the errors you're finding

Reply:
Sure, no worries. On this one, you'll want to keep the existing file, but add an extra role like so:

```yaml
roles:
- district: '78'
  jurisdiction: ocd-jurisdiction/country:us/state:ca/government
  type: lower
  end_date: '2020-12-07'
- jurisdiction: ocd-jurisdiction/country:us/state:ca/place:san_diego/government
  type: mayor
  end_date: '2024-12-31'
```

But you can replace the `contact_details`, `sources`, `links`, and `image` with new data. For sure keep the `other_identifiers` and `id` constant.

Reply:
restored and applied his old id to his new mayor record